Runtime/EOSSDK/SDK/Plugins/x86/xaudio2_9redist.dll.meta

```diff
@@ -11,28 +11,49 @@ PluginImporter:
   isExplicitlyReferenced: 0
   validateReferences: 1
   platformData:
+  - first:
+      : Any
+    second:
+      enabled: 0
+      settings:
+        Exclude Android: 1
+        Exclude Editor: 1
+        Exclude Linux64: 0
+        Exclude OSXUniversal: 0
+        Exclude WebGL: 1
+        Exclude Win: 0
+        Exclude Win64: 0
+        Exclude iOS: 1
+  - first:
+      Android: Android
+    second:
+      enabled: 0
+      settings:
+        AndroidSharedLibraryType: Executable
+        CPU: ARMv7
   - first:
       Any: 
     second:
-      enabled: 1
+      enabled: 0
       settings: {}
   - first:
       Editor: Editor
     second:
       enabled: 0
       settings:
-        CPU: x86
+        CPU: AnyCPU
```

Comment:
Sorry if I'm misunderstanding how the settings work, but couldn't we avoid the "duplicate DLL" issue you mentioned by again splitting by x86 and x64 for the two audio DLLs? Then we could leave Editor here to enabled and not make assumptions about the dev's architecture. Though honestly I'm not sure if these audio DLLs is even required in the editor, I forget if I ever tested that.

Reply:
Ahh, I see what's going on, I'm on Unity 2022.3 and it seems like they removed x86 option completely from the editor settings, since there is no x86 Unity Editor anyway. Here is the topic about this - https://discussions.unity.com/t/dll-cannot-set-cpu-x86-missing-x86-option/943376
Disabling the x86 DLL in the editor completely seems to be the recommended way, so I think it should be fine.

Reply:
Ah gotcha, thanks. It seems they cut off x86 versions starting around Unity 2017 and minimum requirement for OpenEOS is 2020, so this should be fine! Just wanted to make sure we're not alienating more users than before